src/Nokia_LCD.cpp

```diff
@@ -229,14 +229,24 @@ bool Nokia_LCD::printCharacter(char character) {
 bool Nokia_LCD::draw(const unsigned char bitmap[],
                      const unsigned int bitmap_size,
                      const bool read_from_progmem) {
+    
+    return Nokia_LCD::draw(bitmap, bitmap_size, kTotal_columns, read_from_progmem);
+}
```

Comment:
Do you think it makes sense to get rid of the overloaded function by setting `kTotal_columns` as the default value of the third argument?

Reply:
I did it at first, but `kTotal_columns` is in namespace and it's not available on header file.

For instance, I got an error if I do this in Nokia_LCD.h file:  
`bool updateCursorPosition(const unsigned int x_start_position = 0, const unsigned int x_end_position = kTotal_columns);`

But on Nokia_LCD.cpp it works:
`bool updateCursorPosition(const unsigned int x_start_position = 0, const unsigned int x_end_position = kTotal_columns) {
...
}`
The question is, setting default value on implementation (.cpp) is not the standard you are doing. What do you think?

Reply:
I moved the display constants from default namespace to header, so they can share the constants

Reply:
Did you push yet? Can't find the change